app/dashboards/contributor_dashboard.rb

```diff
@@ -9,6 +9,10 @@ class ContributorDashboard < Administrate::BaseDashboard
     last_name: Field::String,
     active: Field::Boolean,
     channels: Field::String.with_options(searchable: false),
+    email: Field::String,
+    username: Field::String,
+    signal_phone_number: Field::String,
+    threema_id: Field::String,
```

Comment:
Could we also add the following attributes?

* `phone`
* `additional_email`
* `telegram_id`
* `zip_code`
* `city`